manifests/params.pp

```diff
@@ -411,15 +411,24 @@
   }
 
   $server_jvm_java_bin      = '/usr/bin/java'
-  $server_jvm_min_heap_size = '2G'
-  $server_jvm_max_heap_size = '2G'
   $server_jvm_extra_args    = '-XX:MaxPermSize=256m'
 
+  # This is some very trivial "tuning". See the puppet reference:
+  # https://docs.puppet.com/puppetserver/latest/tuning_guide.html
+  if $::memorysize_mb >= 2048 {
```

Comment:
This is a string on pre-4.x versions, consider using `0 + $::memorysize_mb` to force the type change.